### ISSUE TITLE
engine is exported as function instead of object

### ISSUE THREAD

**Issue by anandrajneesh** [anandrajneesh/decision-engine#9]
According to docs to use engine below code has to be used and then functions can be invoked on the returned object.
```javascript
const decisionEngine = require('decision-engine')
decisionEngine.addRule(require('./orderSystem/discount.json'))
```
However the decisionEngine is a function instead of object. Functions like addRule and importRules are not available, rather it is required to invoke decisionEngine as a function and then the returned object has the aforementioned functions.
Either docs should mention this or exports should be fixed.

**Comment by anandrajneesh** [comment]
exports should be fixed !

**Comment by anandrajneesh** [comment]
Oddly enough test cases were already invoking _decisionEngine_ as function and then using the returned object.

**Comment by anandrajneesh** [comment]
Fixed in v1.1.4
